index.ts

```diff
@@ -44,6 +45,12 @@ async function main() {
                 // if issue exists in devpool
                 const devpoolIssue = getIssueByLabel(devpoolIssues, `id: ${projectIssue.node_id}`);
                 if (devpoolIssue) {
+                    const issueDetails = await octokit.rest.issues.get({
```

Comment:
`projectIssue = issueDetails`, no need for additional API call, simply use `projectIssue`